PerimeterXModule/PxModule.cs

```diff
@@ -421,13 +452,14 @@ private static string ByteArrayToHexString(byte[] input)
 
         private void HandleVerification(HttpApplication application)
         {
-            bool verified = blockingScore > pxContext.Score;
             PxModuleConfigurationSection config = (PxModuleConfigurationSection)ConfigurationManager.GetSection(PxConstants.CONFIG_SECTION);
+			bool verified = blockingScore > pxContext.Score || config.MonitorMode;
 
-            Debug.WriteLine(string.Format("Request score: {0}, blocking score: {1}, monitor mode status: {2}.", pxContext.Score, blockingScore, config.MonitorMode == true ? "On" : "Off"), PxConstants.LOG_CATEGORY);
+			Debug.WriteLine(string.Format("Request score: {0}, blocking score: {1}, monitor mode status: {2}.", pxContext.Score, blockingScore, config.MonitorMode == true ? "On" : "Off"), PxConstants.LOG_CATEGORY);
 
             if (verified)
             {
+				Debug.WriteLineIf(config.MonitorMode, "Monitor Mode is activitated passing request", PxConstants.LOG_CATEGORY);
```

Comment:
typo - `.. activated. Passing ..`
In general - I'm not sure I understand the purpose of sending a `page_requested` in the case of score > blockingScore while `MonitorMode` is true? Unless this activity represents a "pass" and not only a valid score.